Gulpfile.js

```diff
@@ -140,15 +177,14 @@ gulp.task('watch', function() {
   gulp.watch(['./src/js/**/*.js', 'Gulpfile.js'], ['js'], reload);
 });
 
-gulp.task('build', ['sass', 'images', 'fonts', 'js', 'templates']);
-gulp.task('build:optimized', ['sass:optimized', 'images:optimized', 'fonts', 'js', 'templates:optimized']);
+gulp.task('build', function (cb) {
+  runSequence('clean', ['sass', 'images', 'fonts', 'js', 'templates'], cb);
```

Comment:
What's the difference between `runSequence` and using the built in list with `gulp.task()`?